src/sentry/static/sentry/app/views/settings/components/settingsProjectItem.jsx

```diff
@@ -1,30 +1,28 @@
 import styled from 'react-emotion';
 import React from 'react';
-import createReactClass from 'create-react-class';
 
 import BookmarkStar from 'app/components/projects/bookmarkStar';
 import Link from 'app/components/links/link';
 import ProjectLabel from 'app/components/projectLabel';
 import SentryTypes from 'app/sentryTypes';
 import space from 'app/styles/space';
 
-const ProjectItem = createReactClass({
-  displayName: 'ProjectItem',
-
-  propTypes: {
+class ProjectItem extends React.Component {
+  static propTypes = {
     project: SentryTypes.Project,
     organization: SentryTypes.Organization,
-  },
+  };
 
-  getInitialState() {
-    return {
+  constructor(props) {
```

Comment:
Should we use class props for state and remove the constructor?

Reply:
How would the state derived from props work then?